src/Enums/ticketStates.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace RA\Enums;
+
+abstract class TicketStates
+{
+    public const Closed = 0;
+    public const Open = 1;
+    public const Resolved = 2;
+
+    public static function RenderState(int $type): string
```

Comment:
our functions are mixed PascalCase and camelCase - class methods are usually camelCase - in this case `renderState`. Awesome to see the type hinting here 👌

Reply:
Definitely more of my C# habits creeping in, but immutable things (like constants, function names, etc.) I often denote in PascalCase, making it more obvious when camelCase is used for member variables when camelCase is used. But for the sake of code style and consistency in the codebase, I'll gladly make the change.